alf/algorithms/rl_algorithm.py

```diff
@@ -662,20 +662,25 @@ def _sync_unroll(self, unroll_length: int):
         original_reward_list = []
         initial_state = self.get_initial_rollout_state(self._env.batch_size)
 
+        policy_step_time = 0.
         env_step_time = 0.
         store_exp_time = 0.
         for _ in range(unroll_length):
             policy_state = common.reset_state_if_necessary(
                 policy_state, initial_state, time_step.is_first())
             transformed_time_step, trans_state = self.transform_timestep(
                 time_step, trans_state)
+
+            t0 = time.time()
             policy_step = self.rollout_step(transformed_time_step,
                                             policy_state)
+            policy_step_time += time.time() - t0
 
             action = common.detach(policy_step.output)
 
             t0 = time.time()
-            next_time_step = self._env.step(action)
+            with record_time("time/_sync_unroll/2_env_step"):
```

Comment:
Why is this called "2_env_step"?

Reply:
It indicates that there is the second component within the _sync_unroll block. There should be another summary before this. Added.